tests/yamls/multi-container-pod.yaml

```diff
@@ -5,6 +5,7 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: end-to-end-api-test-pod-multi-container
+  namespace: default
```

Comment:
It's a bit weird adding it to the default ns? Why not use `qa` (and also have the test read from that namespace) like we do with other E2E tests?

Reply:
Apparently because the qa namespace does not exist, changing it back to default